README.md

```diff
@@ -4,10 +4,19 @@
   <a href="https://github.com/actions/setup-ruby"><img alt="GitHub Actions status" src="https://github.com/actions/setup-ruby/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-This action sets up a ruby environment for use in actions by:
+This action sets up Ruby for use in actions by adding to the `PATH` a version of Ruby already existing in the toolcache.
+The toolcache is the set of tools preinstalled in [virtual environments](https://github.com/actions/virtual-environments).
 
-- optionally installing a version of ruby and adding to PATH. Note that this action only uses versions of Ruby already installed in the cache. The action will fail if no matching versions are found.
-- registering problem matchers for error output
+This action currently only supports versions of Ruby in the toolcache.
```

Comment:
We should have a link to each of the readmes (ubuntu 18, mac, windows etc.)

Reply:
Are they not always in sync? It seems very inconvenient for users to have to check 3 links to find whether some version is supported across platforms.

Reply:
No.  And also for each, there's multiple pools globally that roll.  So even the ubuntu 18 one is not correct for days at a time.

Reply:
OK, added a link per OS.